lib/ansible/modules/cloud/amazon/ec2.py

```diff
@@ -16,6 +16,12 @@
 short_description: create, terminate, start or stop an instance in ec2
 description:
     - Creates or terminates ec2 instances.
+    - >
+      Note: This module uses the older boto Python module to interact with the EC2 API.
+      It will still receive bug fixes, but no new features.
```

Comment:
```suggestion
      This module will still receive bug fixes, but no new features.
```

Reply:
Sounds a bit clunky to me - "This module" - "this module" on both lines?

Reply:
The problem is which 'It' we're referring to: boto or the module.

Reply:
How about "M(ec2) will..." instead ?

Reply:
Hmm... in that case it may be better to use `M(ec2)`?
'This module' could also refer to the Python module...

Reply:
Works for me.